src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts

```diff
@@ -178,4 +200,9 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
     this.loadActiveEntrySubscription.unsubscribe();
     this.actionSetDateSubscription.unsubscribe();
   }
+
+  isFeatureToggleActivated(): Observable<boolean> {
+    return this.featureManagerService.isToggleEnabledForUser('update-entries')
+      .pipe(map((enabled: boolean) => enabled));
+  }
```

Comment:
I think this method is easier to delete if it's put into a service. Since you can decouple the logic from the component and when the FT is no longer used you should just remove the service and the references in the component.
What do you think @scastillo-jp ?

Reply:
Oh, It's easier to delete in the test part too.